project1/css/index.css

```diff
@@ -12,104 +12,125 @@ b, u, i, center,
 fieldset, form, label, legend,
 table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, canvas, details, embed, 
-figure, figcaption, footer, header, hgroup, 
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
 menu, nav, output, ruby, section, summary,
 time, mark, audio, video {
-	margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
 }
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
+article, aside, details, figcaption, figure,
 footer, header, hgroup, menu, nav, section {
-	display: block;
+    display: block;
 }
+
 body {
     line-height: 1;
 }
 
-button {
-    background: white;
-    height: 30px;
-    width: 140px;
-    border: 1px solid black;
+header {
+    display: flex;
 }
 
-ol, ul, {
-    display: block;
-    list-style: none;
+nav {
+    width: 85%;
+    height: 100%;
 }
 
-li {
-    display: inline;
-    float: left;
+ol, ul {
+    list-style: none;
+    width: 100%;
+    height: 50px;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: flex-end;
 }
 
-    li a {
-        display: block;
-        padding: 8px 80px;
-        text-decoration: none;
-        color: gray;
-    }
+li a {
+    text-decoration: none;
+    color: gray;
+}
 
-blockquote, q {
-    quotes: none;
+.cta {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 40px;
 }
 
-    blockquote:before, blockquote:after,
-    q:before, q:after {
-        content: '';
-        content: none;
-    }
+.cta-text {
+    display: flex;
+    flex-direction: column;
+    font-size: 80px;
+}
 
-table {
-    border-collapse: collapse;
-    border-spacing: 0;
+button {
+    height: 30px;
+    width: 180px;
+    background: white;
+    border: 1px solid black;
 }
 
-.logo {
-    float: right;
-    padding: 8px 160px;
+.top-content {
+    display: flex;
+    justify-content: center;
+    border-top: 1px solid black;
 }
 
-.cta {
-    display: block;
-    margin: 20px 20px 0 30px;
-    padding: 40px 500px;
+.middle-img {
+    padding: 20px 0 20px 275px;
+    width: 1500px;
 }
-.cta-text {
-    display: inline-block;
-    font-size: 90px;
-    text-align: center;
-    padding: 80px 0 0 0;
+
+
+.top-content .text-content {
+    padding: 20px 0 20px 0;
+    width: 450px;
 }
 
-.top-content {
+.bottom-content {
     display: flex;
+    justify-content: center;
+}
+
+.bottom-content .text-content {
+    padding: 20px 0 20px 0;
+    width: 450px;
+}
+
+.contact {
     border-top: 1px solid black;
-    margin: 20px 20px 20px 20px;
-    padding: 20px 20px 20px 20px;
+    padding: 20px 0 20px 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
 }
 
-.top-content .text-content p {
-    padding:20px 120px;
+footer {
+    margin: 0 auto;
 }
```

Comment:
Everything works as expected. This footer is the only part using margins. The footer can be adjusted to use flex. Good job overall!
Thank you, 
Sara Fritz